README.md

```diff
@@ -79,6 +80,41 @@ descheduler. See the [helm chart README](https://github.com/kubernetes-sigs/desc
 
 The descheduler helm chart is also listed on the [helm hub](https://hub.helm.sh/charts/descheduler/descheduler-helm-chart).
 
+### Install Using Kustomize
+
+You can use the `kustomization.yaml` file as shown in the following example
+- [resources | Kustomize](https://kubernetes-sigs.github.io/kustomize/api-reference/kustomization/resources/)
+
+Run As A Job
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - github.com/kubernetes-sigs/descheduler/kubernetes/job?ref=master
+
+patches:
+  - target:
+      kind: ConfigMap
+      name: descheduler-policy-configmap
+    path: mystrategy.yaml
```

Comment:
If possible every example should be functioning from scratch. Can you also provide `mystrategy.yaml` file as well?

Also, both examples (for job and cronjob) are almost identical. Worth setting both `path` fields to the same value. E.g. `configmap.yaml`.

Reply:
The ability to override configmap is omitted as it is a kustomize feature, not a descheduler feature :bow: